app/models/wordinfo.rb

```diff
@@ -15,21 +15,19 @@ class Wordinfo < ApplicationRecord
   validates_uniqueness_of :word, scope: :lesson, case_sensitive: false
 
   def as_json(options = {})
-    wordinfos =
-      if options.empty? then super({
-        include: [
-          { roots: { only: [:root, :meaning] } },
-          { forms: { only: [:word, :part_of_speech] } },
-          { synonyms: { only: [:word] } },
-          { antonyms: { only: [:word] } },
-          { sentences: { only: [:context_sentence] } }
-        ],
-        except: [:id, :user_id, :lesson_id, :created_at, :updated_at]
-      })
-      else super(options)
-      end
-    wordinfos&.[]('synonyms')&.map! { |synonym| synonym['word'] }
-    wordinfos&.[]('antonyms')&.map! { |antonym| antonym['word'] }
+    wordinfos = super(options.merge(
+      include: [
+        { roots: { only: [:root, :meaning] } },
+        { forms: { only: [:word, :part_of_speech] } },
+        { synonyms: { only: [:word] } },
+        { antonyms: { only: [:word] } },
+        { sentences: { only: [:context_sentence] } }
+      ],
+      except: [:id, :user_id, :lesson_id, :created_at, :updated_at]
+    ))
+    wordinfos&['synonyms']&.map! { |synonym| synonym['word'] }
+    wordinfos&['antonyms']&.map! { |antonym| antonym['word'] }
+    wordinfos&['context_sentence']&.map! { |ct| ct['context_sentence'] }
```

Comment:
I'm really not sure what's suppose to be here, can you just push it up to my branch. It was a merge conflict